src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -824,14 +820,16 @@ public String getDisplayName() {
             return DescriptorVisibilityFilter.apply(null, Jenkins.getInstance().getDescriptorList(TemplateEnvVar.class));
         }
 
+        @SuppressWarnings("unused") // Used by jelly
+        @Restricted(DoNotUse.class) // Used by jelly
+        public Descriptor getDefaultWorkspaceVolume() {
+            return Jenkins.get().getDescriptor(EmptyDirWorkspaceVolume.class);
```

Comment:
```suggestion
        public WorkspaceVolume getDefaultWorkspaceVolume() {
            return WorkspaceVolume.getDefault();
```

as per `dropdownDescriptorSelector.jelly` documentation

```xml
    <st:attribute name="default">
      If specified, this will be chosen as the default value in case the current selection is null. The default can be a specific instance or a descriptor e.g.
      ${descriptor.defaultSettingsProvider} or ${descriptor.defaultSettingsProvider.descriptor}. In the later case, the from input fields will be empty.
    </st:attribute>
```